argo-apps/devspace-provisioning/templates/devworkspace-gitconfig.yaml

```diff
@@ -0,0 +1,16 @@
+{{- range $key := .Values.users }}
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: devworkspace-gitconfig
+  namespace: {{.name}}-dev
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"    
+  labels:
+    app.kubernetes.io/defaultName: git-config-secret
+    app.kubernetes.io/part-of: devworkspace-operator
+    controller.devfile.io/watch-configmap: 'true'
+data:
+  gitconfig: "[filter \"lfs\"]\n    clean = git-lfs clean -- %f\n    smudge = git-lfs smudge -- %f\n    process = git-lfs filter-process\n    required = true\n\n[credential]\n    helper = store --file /.git-credentials/credentials\n\n[user]\n\tname = {{.name}}\n\temail = {{.name}}@che"         
```

Comment:
We know the user's username and password, userN and openshift. Can we inject these credentials into the workspace and confirm that "git push" works without asking for credentials?

Reply:
Done :)